CHANGELOG.md

```diff
@@ -2,7 +2,29 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [3.12.0](https://github.com/sonata-project/SonataCoreBundle/compare/3.11.2...3.13.0) - 2018-11-19
```

Comment:
The diff looks fishy because I made a mistake in the previous release… used 3.13.0 instead of 3.12.0